components/automate-cli/pkg/diagnostics/integration/iam_projects.go

```diff
@@ -162,16 +182,32 @@ func CreateIAMProjectsDiagnostic() diagnostics.Diagnostic {
 				return err
 			}
 
+			loaded := iamProjectSave{}
+			err = tstCtx.GetValue("iam-projects", &loaded)
+			if err != nil {
+				return errors.Errorf(err.Error(), "could not find generated context")
+			}
+
 			tstCtx.SetValue("iam-projects", &iamProjectSave{
-				ID:     projectInfo.Project.ID,
-				RuleID: projectInfo.Project.Rule.ID,
+				ID:      projectInfo.Project.ID,
+				RuleID:  projectInfo.Project.Rule.ID,
+				Skipped: loaded.Skipped,
 			})
 			return nil
 		},
 		Verify: func(tstCtx diagnostics.VerificationTestContext) {
 			loaded := iamProjectSave{}
 			err := tstCtx.GetValue("iam-projects", &loaded)
 			require.NoError(tstCtx, err, "Could not find generated context")
+			if loaded.Skipped == true {
```

Comment:
Can be more concise:
```suggestion
			if loaded.Skipped {
```